subprojects/docs/src/docs/userguide/repositoryTypes.adoc

```diff
@@ -0,0 +1,390 @@
+[[repository_types]]
+== Repository types
```

Comment:
Would you prefer sentence case or title case for titles here?

Reply:
I don't mind using sentence case consistently across the board.